postcommit.py

```diff
@@ -0,0 +1,642 @@
+        diff_fileslist = patch.diffstatdata(util.iterlines(diff_content))
+        return diff_fileslist
+
+ 
+# 	***************************************************************************************************************
+#	diffstat :
+#
+#			Return a files added,removed lines
+#
+#			@param : rev1
+#			@param : rev2
+#			@param : files
+#			
+#			@return :{'AddedLines': 1, 'File': 'testfile', 'RemovedLines': 0}
+#       						
+#	****************************************************************************************************************
+    def diffstat(self, rev1, rev2, files, *pats, **opts):
+	if files:
+        	diff_fileslist = self.difflist(rev1, rev2, files, *pats, **opts)
+		file_stat=[]
+		for x in diff_fileslist:
+			file_numstats = { 'File':[],'AddedLines':[],'RemovedLines':[] }
+			file_numstats['File']=x[0]
+			file_numstats['AddedLines']=x[1]
+			file_numstats['RemovedLines']=x[2]
+			file_stat.append(file_numstats)
+		return file_stat
+	return []
+        
+
+
+class GitRepository():
+
+	def __init__(self, local_repository):
+		self.logs = []
+		self.module_dir = local_repository
+        	self.oldId = oldId
+        	self.newId = newId
+        	self.branch = branch
+        	self.module = os.path.basename(self.module_dir)
+        	self.newchangesets = []
+ 
+# 	***************************************************************************************************************
+#	getCommits :
+#
+#			Initial method in this class to get commits,newchangesets json array
+#
+#			@param : 
+#			
+#			@return : {'commits': [{'files': {'Removed': [], 'Added': [{'AddedLines': '1', 'File': 'test',
+#				   'RemovedLines': '0'}], 'Modified': []}, 'changeset': '3e4983962f67', 'parents': 
+#				   'changed_lines': [1], 'file': 'test'}]}], 'user': 'gomathinayagam.d@zohocorp.com', 
+#				   'branch': 'master', 'push_time': '2018-02-28 17:43:45', 'commit_type': 1, 
+#				   'message': 'test', 'commit_date': '2018-02-28 17:43:21', 'revision': '3e49839'}], 
+#				   'newchangesets': ['3e4983962f6743f1fc55ce53e62553a5f83524c8']}
+#       						
+#	*************************************************************************************************************** 
+
+    	def getCommits(self):
+       	 	self.branchlog(os.path.basename(self.branch),self.oldId,self.newId)
+        	return self.logs, self.newchangesets
+
+	
+#       ***************************************************************************************************************
+#	findNewBranchCommits :
+#
+#			Find number of commits present in inital branch creation
+#
+#			@param : changeset
+#			@param : max_count
+#			@param : skip
+# 			@param : newId
+#			@param : branchname
+#			
+#			@return : boolean       						
+#	*************************************************************************************************************** 
+	def findNewBranchCommits(self,changeset,max_count,skip,newId=None,branchname=None):
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename method "findNewBranchCommits" to match the regular expression ^[a-z_][a-z0-9_]{2,}$. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=python%3AS100)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this parameter "newId" to match the regular expression ^[_a-z][a-z0-9_]*$. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=python%3AS117)